kimtaeheon/spring-crud/src/main/java/com/likelion/springcrud/member/application/MemberService.java

```diff
@@ -0,0 +1,39 @@
+package com.likelion.springcrud.member.application;
+
+import com.likelion.springcrud.member.api.dto.request.MemberSaveRequestDto;
+import com.likelion.springcrud.member.api.dto.response.MemberInfoResponseDto;
+import com.likelion.springcrud.member.api.dto.response.MemberListResponseDto;
+import com.likelion.springcrud.member.domain.Member;
+import com.likelion.springcrud.member.domain.repository.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    @Transactional
+    public void memberSave(MemberSaveRequestDto requestDto) {
+        Member member = Member.builder()
+                .name(requestDto.name())
+                .age(requestDto.age())
+                .build();
+        memberRepository.save(member);
+    }
+
+    public MemberInfoResponseDto memberFindOne(Long id) {
+        Member member = memberRepository.findById(id).orElseThrow(IllegalArgumentException::new);
```

Comment:
요기는 .orElseThrow 부분에 줄 바꿈을 해줌으로써 가독성일 높일 수 있어요.
```suggestion
Member member = memberRepository
       .findById(id)
       .orElseThrow(IllegalArgumentException::new);
```

Reply:
넵!!